Jenkinsfile

```diff
@@ -8,7 +8,7 @@ for (int i = 0; i < (BUILD_NUMBER as int); i++) {
 def branches = [:]
 
 def splits
-if (env.BUILD_NUMBER == '1') {
+if (currentBuild.previousNotFailedBuild == null) {
```

Comment:
well this is disappointing...

![image](https://user-images.githubusercontent.com/494726/97471214-694a2400-1940-11eb-8105-3ca80e3a4e03.png)

Reply:
so `previousNotFailedBuild` is returning builds in progress :(

Reply:
https://github.com/jenkinsci/jenkins/blob/4ff3e8dbf7d3d0cd0619ffefc60d951b3e654c70/core/src/main/java/hudson/model/Run.java#L947-L952

need to iterate on the result of getPreviousCompletedBuild()